lib/new_playground.dart

```diff
@@ -548,45 +562,59 @@ class Playground implements GistContainer, GistController {
     }
   }
 
+  Future showNewFlutter() async {
```

Comment:
Two questions on these three functions:

* Looks like they're all using Layout.dart as the final layout. Is that intentional?
* Could they be collapsed into a single function that took the layout as a parameter and
  - created the sample gist based on that
  - Passed that layout on to _finalizeRoute?

Reply:
That's a good point, I'll make that change